src/select.coffee

```diff
@@ -409,9 +415,9 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
       value = content.objectAt index
     else # setter
       index = content.indexOf value
-      @setHighlightedIndex index, yes
+      @setHighlightedIndex index, @get('shouldEnsureVisible')
```

Comment:
no it won't. This PR is backward compat. The new behavior is only when user override the `shouldEnsureVisible` to false.